package firmament;

message FlowNodeType {

  enum NodeType {
    ROOT_TASK = 0;
    WORKER_TASK = 1;
    UNSCHEDULED = 2;
    CLUSTER_AGGREGATOR = 3;
    RACK_AGGREGATOR = 4;
    COMPUTER = 5;
    SINK = 6;
  }

  required NodeType type = 1;
}